test/parallel/test-fs-write-sync-optional-params.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+require('../common');
+const fs = require('fs');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+
+tmpdir.refresh();
+
+const dest = path.resolve(tmpdir.path, 'tmp.txt');
+const buffer = Buffer.from('zyx');
+
+{
+  let fd = fs.openSync(dest, 'w+');
+
+  assert.throws(() => {
+    fs.writeSync(fd, {}); // Missing buffer, {} is second argument
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE',
+    name: 'TypeError',
+    message: 'The "buffer" argument must be of type string or an instance ' +
+      'of Buffer, TypedArray, or DataView. Received an instance of Object'
```

Comment:
We only test for error code, so the test doesn't fail if the error message is changed in the future. Please only keep the `code` property in the other tests as well.

```suggestion
```

Reply:
Affirmative, removed redundant properties in ac0e126f482f6cf46aa18d695473d88828ebdd2f